src/kixi/event_fixer.clj

```diff
@@ -184,8 +156,20 @@
 (defn validate-new-format-files
   []
   (transduce
-   (map file->error-report)
+   (comp (map file->error-report)
+         (filter (fn [report]
+                   (not (zero? (:error (first (vals report))))))))
    report-errors
    (rest
     (file-seq
      (io/file local-new-format-base-dir)))))
+
+(defn peak-files
```

Comment:
The way you explain this in demo made me think this should be `peek`?

Reply:
lol